REPOSITORIES.md

```diff
@@ -0,0 +1,179 @@
+# Repositories
+
+This document lists all repositories, sorted by their scope, within [Cilium](https://github.com/cilium).
+
+You can read more about the scope and lifecycle of repositories in [REPOSITORY-GUIDELINES.md](https://github.com/cilium/community/blob/main/REPOSITORIES.md).
+
+**Table of Contents**
+
+* [```Core``` Scope](#core-scope)
+* [```Ecosystem``` Scope](#ecosystem-scope)
+* [```Infra``` Scope](#infra-scope)
+* [```Special``` Scope](#special-scope)
+* [```Historical``` Scope](#historical-scope)
+
+## ```Core``` Scope
+
+Learn more about the ```Core``` scope in [REPOSITORY-GUIDELINES.md](https://github.com/cilium/community/blob/main/REPOSITORY-GUIDELINES.md#core).
+
+|   Name  | Description |
+|:--------|:------------|
+| [cilium/charts](https://github.com/cilium/charts) | Helm charts repository for the Cilium project |
+| [cilium/cilium](https://github.com/cilium/cilium) | eBPF-based Networking, Security, and Observability |
+| [cilium/cilium-cli](https://github.com/cilium/cilium-cli) | CLI to install, manage & troubleshoot Kubernetes clusters running Cilium |
+| [cilium/cilium.io](https://github.com/cilium/cilium.io) | Cilium project website |
+| [cilium/ebpf](https://github.com/cilium/ebpf) | A Go library to read, modify and load eBPF programs and attach them to various hooks in the Linux kernel |
+| [cilium/hubble](https://github.com/cilium/hubble) | Network, Service & Security Observability for Kubernetes using eBPF |
+| [cilium/hubble-ui](https://github.com/cilium/hubble-ui) | User interface for [cilium/hubble](https://github.com/cilium/hubble)
+| [cilium/pwru](https://github.com/cilium/pwru) | eBPF-based Linux kernel networking debugger |
+| [cilium/tetragon](https://github.com/cilium/tetragon) | eBPF-based Security Observability and Runtime Enforcement |
+
+## ```Ecosystem``` Scope
+
+Learn more about the ```Ecosystem``` scope in [REPOSITORY-GUIDELINES.md](https://github.com/cilium/community/blob/main/REPOSITORY-GUIDELINES.md#ecosystem).
+
+|   Name  | Description |
+|:--------|:------------|
+| [cilium/arping](https://github.com/cilium/arping) | Go Library to ping a host per arp datagram or query a host mac address
+| [cilium/certgen](https://github.com/cilium/certgen) | A tool to generate and store certificates for Hubble Relay mTLS |
+| [cilium/chaos-testing-examples](https://github.com/cilium/chaos-testing-examples) |Examples of using Cilium for chaos testing and fault injection |
```

Comment:
We haven't updated this repository in 5 years. Do we want to be suggesting that Adopters should use it?

Reply:
I won't go through the rest of these, but we probably want to go through a process to really understand what we're trying to tell adopters about these "Ecosystem" projects and the level of community development on them before we put them in this category.

Reply:
I totally see what you mean. We originally had statuses that helped to indicate to Adopters which repositories were stable (good to use), incubating (use with discretion), or sandbox (not recommended). We received feedback that maintaining 100+ statuses may not be feasible, and I agree. Scope change is much more rare than a status change.

We could differentiate between recommended Ecosystem repositories and those that would fit better under an Incubating or Sandbox status. We could call this new scope```Incubating``` or ```Sandbox```, or something like ```Innovation```.
- Result: ```Core```, ```Ecosystem```, ```Innovation```, ```Internal```, ```Historical``` 

Alternatively, we could expand the meaning of ```Core``` to include all recommended repositories currently listed in Ecosystem. 
- Result: ```Core```, ```Innovation```, ```Internal```, ```Historical```

Reply:
I'd be inclined to advocate for starting as simple as possible to begin with and then build out more process when we find it provides direct benefit.

I understand the motivations for core and historical:
- Core: Flagship repositories where we are actively building communities around them.
- Historical: Repositories where we've explicitly made a decision not to commit further effort.

Given that there are a lot of other repos that don't fit either of these categories, I understand the need for another category that captures the remaining projects. Whether or not we further subdivide that group depends on what we really want to communicate about those projects.

I don't love `Incubating`, `Sandbox` or `Experimental`. I think that these terms make a lot of sense for "Core" projects in a broader ecosystem like CNCF where there is an trajectory through maturity and all successful projects are expected to graduate through the ladder. However, I don't think that they accurately capture the functional needs for individual repos within the Cilium project.

Reply:
A few properties to consider, some inspired from your comment above:
- End-user facing or developer facing
- Generally open to contribution or generally closed to contribution (eg fork)
- Maturity

That said, I also wonder whether we need to split the third group based on some of these properties, or just standardize the way that these properties are described within the individual repos. Presumably this depends on whether we specifically want to introduce some friction between the categories through a centralized repository or not.

Reply:
I agree with keeping things as simple as possible to begin with. Core and Historical make sense as stated above. Then that leaves the rest of the repos that are outside the above. We could combine them all into just something like `Ecosystem` for now and split them out into further categories in the future if we want.

That would indicate the end user facing ones, the ones no longer in use, and "the rest".

Reply:
Done! I combined Internal and Ecosystem and re-wrote the Ecosystem description.